sdk/storage/storage-queue/src/generated/src/storageClientContext.ts

```diff
@@ -11,7 +11,7 @@
 import * as coreHttp from "@azure/core-http";
 
 const packageName = "azure-storage-queue";
-const packageVersion = "12.0.0-preview.5";
+const packageVersion = "12.0.0";
```

Comment:
(Though this is not used) please do the same in blob too for consistency.

Reply:
and file in the other PR.

Reply:
Ideally we don't want to have these constants and the versions. We don't want to change the code, thsi is why i am updating the existing versions

Reply:
If not used then we should remove it

Reply:
@HarshaNalluru Since it is not used, lets not block this PR for it